app/data/census_individual.json

```diff
@@ -1519,6 +1446,11 @@
                                     "value": "English or Welsh"
                                 }
                             }
+                        },
+                        {
```

Comment:
In theory yes, but they get evaluated in order so the 1st would take precedence?